src/App.tsx

```diff
@@ -19,10 +19,31 @@ import useAppStateStore from './store/appStateStore.ts';
 function App() {
     const background = useUIStore((s) => s.background);
     const view = useUIStore((s) => s.view);
+    const setUserMousePosition = useUIStore((s) => s.setUserMousePosition);
+    const setError = useAppStateStore((s) => s.setError);
     const startupInitiated = useRef(false);
     const setSetupDetails = useAppStateStore((s) => s.setSetupDetails);
     const settingUpFinished = useAppStateStore((s) => s.settingUpFinished);
 
+    useEffect(() => {
+        const intervalId = setInterval(() => {
+            invoke('check_user_mouse_position')
```

Comment:
Please add a file like we have in v2: https://github.com/tari-project/tari-universe/blob/main/src/types/invoke.ts

This allows you to type custom handlers in one place

Reply:
Good point, I forgot about that since we last talk about it. Adding it now